src/libcalamares/partition/PartitionQuery.cpp

```diff
@@ -26,6 +27,19 @@ namespace Partition
 using ::Device;
 using ::Partition;
 
+const PartitionTable*
+getPartitionTable( const Partition* partition )
+{
```

Comment:
This is a useful factoring-out, thanks.

Reply:
Feel free to cherry-pick the commit to master.